main-project/src/boards/entity/board.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+// fk없음, entity취합 후 생성예정
+@Entity()
+export class Board extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @Column({ type: 'varchar', length: 255, nullable: false })
+  title: string;
+
+  @Column({
+    type: 'tinyint',
+    width: 1,
+    default: false,
+    comment: '인원 모집 여부',
+    nullable: false,
+  })
+  done: boolean;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  description: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  location: string;
+
+  @Column({ type: 'date', nullable: true })
+  time: Date;
+
+  @CreateDateColumn()
+  created_date: Date;
+
+  @UpdateDateColumn()
+  updated_date: Date;
+
+  @DeleteDateColumn({ nullable: true })
+  deleted_date: Date;
```

Comment:
```suggestion
  @CreateDateColumn({ name: 'created_date' })
  createdDate: Date;

  @UpdateDateColumn({ name: 'updated_date' })
  updatedDate: Date;

  @DeleteDateColumn({ name: 'deleted_date', nullable: true })
  deletedDate: Date;
```
DB 컬럼명은 스네이크 케이스를 주로 사용하지만 코드 내에서는 카멜 케이스를 지향합니다. 컬럼 내부에 여러 옵션이 있으니 공식 문서를 확인해보는걸 추천해요 ㅎㅎ

Reply:
앗 제가 실수를 했네요ㅎㅎ 감사합니다 :)